pkgs/applications/networking/browsers/yandex-browser/default.nix

```diff
@@ -50,11 +50,11 @@
 
 stdenv.mkDerivation rec {
   pname = "yandex-browser";
-  version = "21.6.2.817-1";
+  version = "21.9.1.600-1";
 
   src = fetchurl {
     url = "http://repo.yandex.ru/yandex-browser/deb/pool/main/y/${pname}-beta/${pname}-beta_${version}_amd64.deb";
```

Comment:
Returns a 404. Shall we upload this to web.archive.org?

Reply:
> Returns a 404. Shall we upload this to web.archive.org?

The new version came out. They do not leave old links, immediately change the main one. Right now I'll fix everything.

Reply:
As for Wayback, I have long wanted to do it for a long time, but it did not work out.
![image](https://user-images.githubusercontent.com/86075850/147232739-dfd84ea4-d2f8-436b-ab90-44b7436b1f05.png)